frontend/src/pages/groups/GroupList.tsx

```diff
@@ -31,15 +32,16 @@ const COLUMNS = [
   },
   {
     Header: 'Location',
-    accessor: 'location',
+    accessor: (group: Group) =>
+      `${group.primaryLocation.townCity} (${group.primaryLocation.countryCode})`,
```

Comment:
I realized that, instead of mapping over the data manually, we could format the content of the cells in this configuration. Pretty neat feature!

Reply:
Ah that's nice yeah!